src/main/scala/devsearch/features/Features.scala

```diff
@@ -5,6 +5,7 @@ import devsearch.parsers._
 
 abstract class Feature(position: CodeFilePosition) extends java.io.Serializable {
   def key: String
+  override def toString() = key + "," + position.toString + ";"
```

Comment:
Oh actually I just realized that keys could have "," characters inside them (for example if we have a key that stores a list of arguments)! These need to be escaped otherwise the splitting won't work. This could also happen in the file name, maybe (not sure if github has limitations about this).